sys/include/usb/usbus/msc.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019-2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ *
+ *
+ */
+
+/**
+ * @ingroup     usbus_msc
+ *
+ * @{
+ *
+ * @file
+ * @brief       USBUS Mass Storage Class functions definitions
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef USB_USBUS_MSC_H
+#define USB_USBUS_MSC_H
+
+#include <stdint.h>
+#include "usb/usbus.h"
+#include "usb/usbus/msc/scsi.h"
+#include "mtd.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef USBUS_MSC_EXPORTED_DEV
+#define USBUS_MSC_EXPORTED_DEV MTD_NUMOF
+#endif /* USBUS_MSC_EXPORTED_DEV */
+
+typedef enum {
+    WAITING,
+    WAIT_FOR_TRANSFER,
+    DATA_TRANSFER_IN,
+    DATA_TRANSFER_OUT,
+    DATA_TRANSFER_LAST,
+    GEN_CSW
+} usbus_msc_state_t;
```

Comment:
Eh, doxygen is right that those should be documented.
Someone (or you yourself) touching this code a few years down the line will be thankful to have an explanation what those states mean.

Reply:
Added comments and remove `DATA_TRANSFER_LAST` which was unused. Probably some leftover from long ago.